packages/sdk-components-react/src/vimeo-spinner.tsx

```diff
@@ -0,0 +1,24 @@
+import {
+  forwardRef,
+  type ElementRef,
+  type ComponentProps,
+  useContext,
+} from "react";
+import { Box, defaultTag } from "./box";
+import { VimeoContext } from "./vimeo";
+
+type Props = ComponentProps<typeof Box>;
+
+export const VimeoSpinner = forwardRef<ElementRef<typeof defaultTag>, Props>(
+  (props, ref) => {
+    const vimeoContext = useContext(VimeoContext);
+
+    if (vimeoContext.status !== "initialized") {
+      return null;
+    }
+
+    return <Box {...props} ref={ref} />;
```

Comment:
Do we even need it to be Box?

Reply:
Just a div would work. We don't need it to be semantic.

Reply:
true